Healthy/Classes/Modules/Dashboard/DashboardViewController.swift

```diff
@@ -33,6 +33,54 @@ final class DashboardViewController: UIViewController {
 
         configureView()
     }
+
+    // MARK: - Function for header section creation
+    func makeHeaderSection(title: String, content: UIView) -> UIView {
+
+        let titleLabel = UILabel()
+        titleLabel.text = title
+
+        let stack = UIStackView(arrangedSubviews: [titleLabel, content ])
+        view.addSubview(stack)
+        stack.backgroundColor = .darkGray
+
+        stack.axis = .horizontal
+        stack.alignment = .center
+        stack.distribution = .fillEqually
+       stack.spacing = 10
+
+        stack.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            stack.leadingAnchor.constraint(equalTo:
+                                            view.safeAreaLayoutGuide.leadingAnchor,
+                                           constant: 8),
+            stack.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor,
+                                            constant: -8),
+            stack.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 8),
+            stack.heightAnchor.constraint(equalToConstant: view.bounds.height*0.1)
+        ])
+
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            titleLabel.leadingAnchor.constraint(equalTo: stack.leadingAnchor, constant: 8),
+            titleLabel.trailingAnchor.constraint(equalTo: content.leadingAnchor),
+            titleLabel.topAnchor.constraint(equalTo: stack.topAnchor),
+            titleLabel.bottomAnchor.constraint(equalTo: content.topAnchor, constant: -8)
+        ])
+
+        content.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            content.leadingAnchor.constraint(equalTo: titleLabel.trailingAnchor, constant: 10),
+            content.topAnchor.constraint(equalTo: stack.topAnchor),
+            content.trailingAnchor.constraint(equalTo: stack.trailingAnchor, constant: -8),
+            content.bottomAnchor.constraint(equalTo: content.topAnchor, constant: -8)
+        ])
+
+        return stack
+    }
```

Comment:
It's a good practice to select all and indent the code whenever you finish writing code in a file, or you can download SwiftFormat to do the formatting for you.

```suggestion
    func makeHeaderSection(title: String, content: UIView) -> UIView {
        let titleLabel = UILabel()
        let stack = UIStackView(arrangedSubviews: [titleLabel, content])

        stack.translatesAutoresizingMaskIntoConstraints = false
        titleLabel.translatesAutoresizingMaskIntoConstraints = false
        content.translatesAutoresizingMaskIntoConstraints = false

        titleLabel.text = title
        view.addSubview(stack)
        stack.backgroundColor = .darkGray
        stack.axis = .horizontal
        stack.alignment = .center
        stack.distribution = .fillEqually
        stack.spacing = 10

        NSLayoutConstraint.activate([
            stack.leadingAnchor.constraint(
                equalTo: view.safeAreaLayoutGuide.leadingAnchor,
                constant: 8),
            stack.trailingAnchor.constraint(
                equalTo: view.safeAreaLayoutGuide.trailingAnchor,
                constant: -8),
            stack.topAnchor.constraint(
                equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 8),
            stack.heightAnchor.constraint(equalToConstant: view.bounds.height*0.1)
        ])

        NSLayoutConstraint.activate([
            titleLabel.leadingAnchor.constraint(equalTo: stack.leadingAnchor, constant: 8),
            titleLabel.trailingAnchor.constraint(equalTo: content.leadingAnchor),
            titleLabel.topAnchor.constraint(equalTo: stack.topAnchor),
            titleLabel.bottomAnchor.constraint(equalTo: content.topAnchor, constant: -8)
        ])

        NSLayoutConstraint.activate([
            content.leadingAnchor.constraint(equalTo: titleLabel.trailingAnchor, constant: 10),
            content.topAnchor.constraint(equalTo: stack.topAnchor),
            content.trailingAnchor.constraint(equalTo: stack.trailingAnchor, constant: -8),
            content.bottomAnchor.constraint(equalTo: content.topAnchor, constant: -8)
        ])

        return stack
    }
}
```